packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -130,24 +130,31 @@ export const setQueryTaggingInfo = moduleDebounce(
  *
  * @public
  */
-export const trackResultClickedWire = createTrackResultWire('click');
+export const trackResultClickedWire = createTrackWire('click');
+
+/**
+ * Tracks the tagging of the banner.
+ *
+ * @public
+ */
+export const trackBannerClickedWire = createTrackWire('click');
 
 /**
  * Performs a track of a result added to the cart.
  *
  * @public
  */
-export const trackAddToCartWire = createTrackResultWire('add2cart');
+export const trackAddToCartWire = createTrackWire('add2cart');
 
 /**
- * Factory helper to create a wire for the track of a result.
+ * Factory helper to create a wire for the track of a taggable element.
  *
  * @param property - Key of the tagging object to track.
- * @returns A new wire for the given property of the result tagging.
+ * @returns A new wire for the given property of the taggable element.
  *
  * @internal
  */
-function createTrackResultWire(property: keyof Tagging): Wire<Result> {
+function createTrackWire(property: keyof Tagging): Wire<Taggable> {
```

Comment:
Could we export this function to be used outside X? At Motive, we also track things like clicks in suggestions or when the user opens the search-layer. We have a copy of this function to create wires for tacking that. It would be awesome if we could use this instead of a copy.